meadow-client/test/Blockly/Headless.js

```diff
@@ -0,0 +1,22 @@
+/*eslint-env node*/
+'use strict';
+
+exports.createBlocklyInstance_ = function() {
+    return require('node-blockly');
+}
+
+exports.createWorkspace_ = function(blockly) {
+    return new blockly.Workspace;
+}
+
+exports.initializeWorkspace_ = function(blockly, workspace) {
+    var xmlText = '<xml id="workspaceBlocks" style="display:none"><block type="contract" x="13" y="187" id="root_contract"></block></xml>';
```

Comment:
I remain uncomfortable with this stringyness.

Reply:
I'm not convinced, apart from the fact I can't work out how to create elements in a node env yet it would be something like:
```javascript
    var workspaceBlocks = document.createElement('xml');
    workspaceBlocks.setAttribute('id', 'workspaceBlocks');
    workspaceBlocks.setAttribute('style', 'display:none');
    var block = document.createElement('block');
    block.setAttribute('id', 'root_contract');
    block.setAttribute('type', 'contract');
    block.setAttribute('x', '13');
    block.setAttribute('y', '187');
    workspaceBlocks.appendChild(block);
```

Personally I find this much more difficult to understand.

Reply:
I prefer the latter. Feels less brittle. But I don't feel strongly about it. Final call's yours to make. 

_makes universal gesture for "après vous monsieur"_

Reply:
leaving as is 😛